a-trust-connector-demo/src/ProcessHelper.java

```diff
@@ -0,0 +1,10 @@
+import ch.ivyteam.ivy.cm.ContentObjectValue;
```

Comment:
please move this class to a package. working with default package is not good.

and rename this class "ProcessHelper" is too generic - maybe something with cms.

the method getTrustTestDoc is doing just generic stuff. but the method name is specific.